klog.go

```diff
@@ -585,9 +591,10 @@ func (l *loggingT) header(s severity, depth int) (*buffer, string, int) {
 		file = "???"
 		line = 1
 	} else {
-		slash := strings.LastIndex(file, "/")
-		if slash >= 0 {
-			file = file[slash+1:]
+		dir := path.Base(path.Dir(file))
+		file = path.Base(file)
```

Comment:
`path.Base(path.Dir(...))` is significantly more expensive than the current code. until we have performance tests in place that tell us whether we are regressing (as several recent functional additions did), I don't think we should merge this

Reply:
That's fine. If we can find it useful we can keep it until we have a better solution in place.